extensions/apiconnect/src/types/oas-extensions/x-ibm-configuration/assembly/execute/rate-limit.policy.ts

```diff
@@ -0,0 +1,27 @@
+export type RateLimitPolicy = DataPowerAPIGateway;
+
+export type DataPowerAPIGateway = V200;
+
+export type V200 = {
+  version: string;
+  title?: string;
+  description?: string;
+  source: 'catalog-named' | 'plan-named' | 'gateway-named' | 'plan-default';
+  operation?: 'consume' | 'replenish' | 'inc' | 'dec'; // No clear documentation on this.
```

Comment:
The documentation is unclear. It seems like there can be multiple limit types simultaneously (i.e. There must be any combination of `rate-limit`, `burst-limit`, and `count-limit`), but yet the default value of this is dependent on which limit is used.

From https://www.ibm.com/docs/en/api-connect/10.0.x?topic=execute-ratelimit:
> You must provide at least one rate, burst, or count limit..

Initially, I assumed `operation` was supposed to be defined inside of either `rate-limit`, `burst-limit` or `count-limit`, but I was not able to find any info on this on the linked docs.